src/Momento.Sdk/Internal/Middleware/HeaderMiddleware.cs

```diff
@@ -11,9 +11,9 @@ namespace Momento.Sdk.Internal.Middleware
 {
     internal class Header
     {
-        public const string AuthorizationKey = "Authorization";
-        public const string AgentKey = "Agent";
-        public const string RuntimeVersionKey = "Runtime_Version";
```

Comment:
@bruuuuuuuce any idea why this was underscored?

Reply:
was either old or must have been a typo, we have aligned with backend team that we want to use `runtime-version`